modules/core/src/main/resources/reference.conf

```diff
@@ -1,4 +1,11 @@
 {
+
+  "gcpUserAgent": {
+    "productName": "Snowplow OSS"
+    "productName": ${?GCP_USER_AGENT_PRODUCT_NAME}
+    "productVersion": "lake-loader"
```

Comment:
Seems like `productVersion` isn't used anywhere ?

Reply:
In the gcp `application.conf` file [here](https://github.com/snowplow-incubator/snowplow-lake-loader/blob/927b799f5c2713dc35b446ff2fb047088f16503c/modules/gcp/src/main/resources/application.conf#L4) I copy this `gcpUserAgent` block into the pubsub source/sink configurations.  Then the common-streams implementations expects to see a `productVersion` field and it is used [here](https://github.com/snowplow-incubator/common-streams/blob/bc6efa7494d4699c80c63e34782c75b563239286/modules/pubsub/src/main/scala/com/snowplowanalytics/snowplow/pubsub/GcpUserAgent.scala#L22).

Sorry I know that is really not obvious.